Sources/Hooks/Internals/HookDispatcher.swift

```diff
@@ -17,6 +17,13 @@ internal final class HookDispatcher: ObservableObject {
         }
     }
 
```

Comment:
Now that the clear method has been added, deinit should be removed, right?
The execution time of deinit is not controllable.

This would be an important improvement ~